pkg/rpc/client/rpc.go

```diff
@@ -774,53 +775,40 @@ func (c *Client) CalculateValidUntilBlock() (uint32, error) {
 }
 
 // AddNetworkFee adds network fee for each witness script and optional extra
-// network fee to transaction. `accs` is an array signer's accounts.
+// network fee to transaction. `accs` is an array of signer's accounts with
+// matching order.
```

Comment:
We better not touch it for now (move to separate PR for discussion).

Reply:
#1863.